src/routes/(app)/admin[[hash=admin_hash]]/nostr/+page.svelte

```diff
@@ -86,7 +86,19 @@
 		<li>{relay}</li>
 	{/each}
 </ul>
-
+<h2 class="text-2xl">Intro Message</h2>
```

Comment:
Add text to specificy this is the message sent when receiving a message that doesn't match a command.

Reply:
Yes, nice idea.
If we have !unknowncommand, we can trigger the !help answer.